libs/core/tests/unit_tests/runnables/__snapshots__/test_graph.ambr

```diff
@@ -1040,7 +1040,7 @@
   	PromptTemplate(PromptTemplate)
   	FakeListLLM(FakeListLLM)
   	Parallel_as_list_as_str_Input(Parallel<as_list,as_str>Input)
-  	Parallel_as_list_as_str_Output([Parallel_as_list_as_str_Output]):::last
+  	Parallel_as_list_as_str_Output([Parallel<as_list,as_str>Output]):::last
```

Comment:
why is this one changing?

Reply:
cause we used to do use key as display name for the last node, and we apply `_escape_node_label` on the key, but now since we're passing value explicitly, it's no longer escaped (which i think is actually better)